components/CustomError.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Paper, Alert, Button } from '@mui/material'
+
+const CustomError = ({ title, message }) => {
```

Comment:
Let's keep the code consistent, so use `function CustomError` instead of `const`. A good practice is looking at similar files to see how the other developers wrote the code. You can look inside the `components` folder.